kbase-extension/static/kbase/css/editorCell.css

```diff
@@ -176,12 +159,39 @@ control with padding, and then scooting the icon back into its column.
   display: none;
 }
 
-.kb-elapsed-time {
-  font-family: monospace;
+.kb-editor-cell [data-element=run-control-panel] {
+  width: 100%;
+}
+
+.kb-editor-cell [data-element=tab-pane] {
+  border-top: 2px solid rgb(33 150 243);
 }
 
-.kb-elapsed-time.-active {
-  color: lime;
+.kb-editor-cell [data-element=tab-pane] > div:empty {
+  padding: 0;
+}
+
+.kb-editor-cell [data-element=tab-pane] > div {
+  padding: 4px;
+}
+
+.kb-editor-cell .btn-primary.kb-btn-action.-rerun {
+  color: rgb(33 150 243);
```

Comment:
If we can, I think it would be a good idea just to stick to hex for colors to be consistent. I'm not sure if changing all of those values is in scope of this ticket though.

Reply:
I was thinking of extracting all colours and creating a new scss colour palette file, and then replacing all the existing values with variables from that file. It would probably be a good idea for me to re-run the colour converter and move everything over to hex first.

Reply:
I've moved everything over to RGB instead.